firestore/FirestoreSwiftUIExample/Firestore+Extension.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Firestore+Extension.swift
+//  FirestoreSwiftUIExample
+//
+
+import Firebase
+
+extension Firestore {
+  func populate() {
+    let words = ["Bar", "Fire", "Grill", "Drive Thru", "Place", "Best", "Spot", "Prime", "Eatin'"]
+
+    let cities = Restaurant.cities
+    let categories = Restaurant.categories
+
+    for _ in 0 ..< 20 {
+      let randomIndexes = (Int(arc4random_uniform(UInt32(words.count))),
+                           Int(arc4random_uniform(UInt32(words.count))))
+      let name = words[randomIndexes.0] + " " + words[randomIndexes.1]
+      let category = categories[Int(arc4random_uniform(UInt32(categories.count)))]
+      let city = cities[Int(arc4random_uniform(UInt32(cities.count)))]
+      let price = Int(arc4random_uniform(3)) + 1
+      let photo = Restaurant.imageURL(forName: name)
+
+      // Basic writes
+
+      let collection = self.collection("restaurants")
+
+      let restaurant = Restaurant(
+        name: name,
+        category: category,
+        city: city,
+        price: price,
+        ratingCount: 10,
+        averageRating: 0,
+        photo: photo
+      )
+
+      let restaurantRef = collection.document()
+      do {
+        try restaurantRef.setData(from: restaurant)
+      } catch {
+        fatalError("Encoding Restaurant failed: \(error)")
+      }
+
+      let batch = self.batch()
+      //TODO: guard let user = Auth.auth().currentUser else { continue }
```

Comment:
TODO or done?

Reply:
Ah, I marked these at TODO because I haven't set up Auth. I'll remove for this PR.